### PR TITLE
Feature/287 redesign filtermenu

### PR DESCRIPTION
**Pull request by juliaslaats**
## Description

Implemented new design of filtermenu as designed in #413 
@waltersajtos helped with changing the sorting-options from two menu's to one. 

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Go to project overviewpage
2. See new filtermenu
3. Check that it looks the same as the designs

## Link to issue

Closes: #287

### CODE REVIEW

File: src/app/modules/project/overview/overview.component.ts [hunk 1]
```diff
@@ -309,27 +280,14 @@ export class OverviewComponent implements OnInit, AfterContentInit {
...
-    const convertedIndex = +value;
-    const foundOption = this.highlightSelectOptions[convertedIndex];
-    if (foundOption == null) {
-      return;
-    }
-    this.currentOnlyHighlightedProjects = foundOption.value;
+    this.currentSortType = value.split(',')[0];
+    this.currentSortDirection = value.split(',')[1];
+    console.log(`value.type: ${this.currentSortType}`);
+    console.log(`value.direction: ${this.currentSortDirection}`);
```
Comment (Ruby77) [inline src/app/modules/project/overview/overview.component.ts:290]: Is this something that should get logged?

**Review by Ruby77 — CHANGES_REQUESTED** [review]
Nice implementation.

Noticed small difference in the design, so would like to ask if this change is intended, or not :) 

image
If I decrease my zoom on the page, the filter bar doesn't stay on the right side of the page.

Furthermore, I noticed the selection drop down in the filter menu, looks a little different on Firefox (in my opinion not as nice as in Chrome). Is it possible to change this design?
image

Further nothing to criticize. Good work :fire:

File: src/app/modules/project/overview/overview.component.ts (hunk 1 above)
Comment (juliaslaats) [reply src/app/modules/project/overview/overview.component.ts]: Deze heb ik er uit gehaald :)

**Review by Ruby77 — CHANGES_REQUESTED** [review]
I noticed that the amount of project cards on one row changed from 4 to 5 projects. I would like to know why you decided to change this?

In the previous version, with 4 projects on one row, all rows are always completely occupied, so there was never 3 projects on a row where 4 projects could appear.

But currently, with 5 projects on one row, not all rows are always completely occupied.
image

In my opinion it's better to fill all rows, but please discuss this with @liekevdvoort.

**Comment by waltersajtos** [comment]
We decided to go with for because it worked on all screen sizes, it would mean that the cards will need to get bigger tho. If you don't make the cards bigger you will get too much white space I think

**Comment by juliaslaats** [comment]
@Ruby77 
> I noticed that the amount of project cards on one row changed from 4 to 5 projects. I would like to know why you decided to change this?

I didn't decide to change this, I think this is included in the responsiveness of the grid. I'm not fully sure how to fix that, besides changing the size of the filtermenu to be bigger. 

I think @liekevdvoort won't have time to look at this until tonight, but maybe she has a better idea on how to fix this

**Comment by liekevdvoort** [comment]
It has to do with the container div beeing bigger now

**Comment by juliaslaats** [comment]
> It has to do with the container div beeing bigger now

It needed to be bigger for the menu to be all the way to the left... I can have another look at it, see if I can fix that another way

**Comment by juliaslaats** [comment]
@Ruby77 @liekevdvoort  I added the max-with I removed in another place, so now it will get a max. of 4 cards on each row, as it did before.

**Review by Ruby77 — APPROVED** [review]
Looks good. Thanks Julia!

**Review by Brend-Smits — APPROVED** [review]
Great work :) Thanks for the swift implementation @juliaslaats & design @liekevdvoort

**Pull request merged**